bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="androidtv"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="shieldtv">
+
+		<label>ShieldTV</label>
+		<description>Nvidia ShieldTV</description>
+
+		<channels>
+			<channel id="debug" typeId="debug"/>
+			<channel id="keypress" typeId="keypress"/>
+			<channel id="keyboard" typeId="keyboard"/>
+			<channel id="keycode" typeId="keycode"/>
+			<channel id="pincode" typeId="pincode"/>
+			<channel id="app" typeId="app"/>
+			<channel id="appname" typeId="appname"/>
+			<channel id="appurl" typeId="appurl"/>
+			<channel id="player" typeId="player"/>
+			<channel id="power" typeId="system.power"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+		</channels>
+
+		<properties>
+			<property name="Device Name">unknown</property>
+			<property name="Device ID">unknown</property>
+			<property name="Architectures">unknown</property>
+			<property name="Manufacturer">unknown</property>
+			<property name="Model">unknown</property>
+			<property name="Android Version">unknown</property>
+			<property name="Remote Server">unknown</property>
+			<property name="Remote Server Version">unknown</property>
```

Comment:
Please use camel-case for the property names, see:

https://next.openhab.org/docs/developer/guidelines.html#naming-convention